bats/tests/common-setup.bash

```diff
@@ -21,8 +21,12 @@ _common_setup_file() {
   fi
   # clone or update the sources
   if [ -d $PYRSIA_TEMP_DIR/.git ]; then
-    git --git-dir=$PYRSIA_TEMP_DIR/.git fetch
-    git --git-dir=$PYRSIA_TEMP_DIR/.git --work-tree=$PYRSIA_TEMP_DIR merge origin/$GIT_BRANCH
+    export GIT_DIR=$PYRSIA_TEMP_DIR/.git
```

Comment:
I ended up adopting this because I realized it is not too complicated.

Reply:
If you're confident this solution works then I think we can go ahead and update this part.